stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Android.kt

```diff
@@ -20,4 +23,17 @@ class Android {
             viewModel.bindView(channelListHeaderView, viewLifecycleOwner)
         }
     }
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/android/message_input_view_neo">Message Input View</a>
+     */
+    inner class MessageInput(private val messageInputView: MessageInputView) : Fragment() {
```

Comment:
Why is this class `inner`?

Reply:
We do that in other places in the docs so that we can access properties of the outer class, possibly not needed here (but generally just doesn't really matter)

Reply:
Just following the pattern, but let's remove it.